rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/ParishService.java

```diff
@@ -1,12 +1,11 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.parish.domain;
 
-import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
+import static java.util.Objects.requireNonNull;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
-
-import static java.util.Objects.requireNonNull;
+import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
```

Comment:
\[Checkstyle\] WARNING: 'pl\.oaza\.warszawa\.dor\.rekolekcje\.api\.parish\.dto\.ParishDTO' should be separated from previous import group by one line\.